python/cudf/cudf/core/dataframe.py

```diff
@@ -7339,9 +7266,16 @@ def pct_change(
             Periods to shift for forming percent change.
         fill_method : str, default 'ffill'
             How to handle NAs before computing percent changes.
+
+            .. deprecated:: 23.12
```

Comment:
We probably need to update these since 23.12 was released in the past.
```suggestion
            .. deprecated:: 24.04
```